src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -586,6 +586,8 @@ private function getArgumentsAsPhp(\DOMElement $node, string $name, string $file
                     $arguments[$key] = new ServiceLocatorArgument($arg);
                     break;
                 case 'tagged':
+                    trigger_deprecation('symfony/dependency-injection', '7.2', 'Type "%s" of tag <%s> in "%s" is deprecated, use "tagged_iterator" or "tagged_locator" instead.', $arg->getAttribute('type'), $name, $file);
```

Comment:
the replacement doing the same thing is `tagged_iterator` (see this `$forLocator` variable that is used to handle differences for `tagged_locator`)

Reply:
I was wondering whether indicating both was not a good thing for discovery, but it's actually more accurate to just put the iterator in.